### PR TITLE
BZ#1916374 Approving CSRs is two step process

### LINKED ISSUE #27716: [enterprise-4.5] Issue in file installing/installing_bare_metal/installing-bare-metal.adoc
### Which section(s) is the issue in?
https://docs.openshift.com/container-platform/4.6/installing/installing_bare_metal/installing-bare-metal.html#installation-approve-csrs_installing-bare-metal

### What needs fixing?
It's a very small issue with the order of operations.  In UPI baremetal deployments, you won't see any new nodes until you sign the CSR's.  The documentation implies that you'll see the nodes, and then need to sign CSR's AFTER you're able to see them with "oc get nodes".

When deploying a new cluster, I only saw my master nodes with "oc get nodes".  Only after signing all pending CSRs was I able to list the worker nodes that I needed to add.

This lead me to believe that my new nodes were not deployed correctly, when in fact, they were available all along.

### PR DESCRIPTION
**Pull request by codyhoag**
https://bugzilla.redhat.com/show_bug.cgi?id=1916374

### CODE REVIEW

**Comment by codyhoag** [comment]
Preview: https://deploy-preview-29143--osdocs.netlify.app/openshift-enterprise/latest/installing/installing_vsphere/installing-vsphere.html#installation-approve-csrs_installing-vsphere

**Comment by codyhoag** [comment]
@gpei PTAL

**Comment by gpei** [comment]
LGTM, thanks for the update.

**Comment by lbarbeevargas** [comment]
One comment, but otherwise LGTM! Because this BZ is requesting that the CSR approval process be spelled out for the two steps (client then server requests), should that two step process be mentioned in the module's introduction?

**Comment by codyhoag** [comment]
@lbarbeevargas yeah, I was on the fence about this. I do think that would make it clearer, so I'll add a quick note on it :+1:

**Comment by lbarbeevargas** [comment]
> @lbarbeevargas yeah, I was on the fence about this. I do think that would make it clearer, so I'll add a quick note on it 👍

Cool, yeah just gives the user a heads up before jumping into the procedure. Especially because the procedure is a running list and not broken up into two distinct parts for client and server requests.

File: modules/installation-approve-csrs.adoc [hunk 1]
```diff
@@ -31,7 +31,7 @@ endif::[]
 [id="installation-approve-csrs_{context}"]
 = Approving the certificate signing requests for your machines
 
-When you add machines to a cluster, two pending certificate signing requests (CSRs) are generated for each machine that you added. You must confirm that these CSRs are approved or, if necessary, approve them yourself.
+When you add machines to a cluster, two pending certificate signing requests (CSRs) are generated for each machine that you added. You must confirm that these CSRs are approved or, if necessary, approve them yourself. The client requests must be approved first, followed by the server requests.
```
Comment (lbarbeevargas) [inline modules/installation-approve-csrs.adoc:34]: Small intro addition looks good! 🚀

**Pull request merged**